hackaton.iml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I ja zadaję tutej ogólne pytanie - a czemóż to czemóż iml do projektu dodajemy?

Reply:
Też się teraz nad tym zastanawiam :D

Reply:
kasa2001 zaczął :P Na masterze jest już iml i spójrzcie czyj :D

Reply:
Spadaj. :D